modules/ROOT/partials/configuration/improved_custom_elements.adoc

```diff
@@ -0,0 +1,66 @@
+[[improved-custom-elements]]
+== Improved Custom Elements
+
+Enhancements have been made to the existing custom_elements option with the introduction of the `addCustomElements` API in {productname}. This improvement refines the structure of custom elements to accommodate more complex configurations, enabling better customization for users.
+
+* **New Custom Element Structure:** The custom_elements option and the `addCustomElements` API now support a more complex structure. This structure is defined by a record where the key represents the name of the element, and the value corresponds to a detailed specification.
+* **Attributes and Inheritance:** Elements can now inherit attributes and children from other specified elements using the extends property in the Custom Element Spec.
+* **Attribute and Children Specifications:** The Custom Element Spec includes properties such as attributes, which lists attribute names or presets like `@global`, and `children`, which lists child element names or presets like `block`, `phrasing`, or `flow`.
```

Comment:
```suggestion
* **Attribute and Children Specifications:** The Custom Element Spec includes properties such as `attributes` which lists attribute names, or `children` which lists valid child element names. The `children` property includes presets like `@global`, `@blocks`, `@phrasing`, or `@flow`.
```

Reply:
i think this change better describes what is available in the spec. @spocke can you verify the correctness of this statement?